kedro-org/kedro-viz
# Fix- Flowchart does not load for static data inputs

## Description

fixes[ issue 840](https://github.com/kedro-org/kedro-viz/issues/840). 

Currently the flowchart would not load for static data soures, which causes problems for our npm component usecase. This PR fixes that. 

## Development notes

This problem is mainly due to the fact that the `getGraphInput` action does not get dispatched on first load ( it is only called via web worker which will only be triggered for future state changes. ) I have added an action within the `configureStore` method to ensure this action still gets fired on first load. 

I have also modified the test set up with mocking the `configureStore` function to ensure it properly configures the store. 

Please note that I had not added any tests for this given the lack of a test suite setup for `store.js`, and it might be an overkill to add that just for testing the dispatch of this action( especially this is a quick fix with the aim to be included in the release tomorrow). 

That said, happy to look into adding one for `store.js` in general in a future ticket. 

## QA notes

Try loading the app via one of the static data sources - we should now be able to load the flowchart properly. 

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/843"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
